tests/helpers/test_reports_helpers.py

```diff
@@ -66,22 +66,25 @@ def test_delete_reports(app, freezer):
     for filename in filenames:
         assert os.path.isfile(f"{app.config['REPORTS_DIR']}/{filename}") == False
 
+
 def test_get_cherrypicked_samples(app, freezer):
 
-  expected = pd.DataFrame(['MCM001', 'MCM003', 'MCM005'], columns=['Root Sample ID'], index=[0, 1, 2])
-  samples = ['MCM001','MCM002','MCM003','MCM004','MCM005']
-
-  with app.app_context():
-    with patch(
-      "sqlalchemy.create_engine", return_value=Mock()
-    ):
-      with patch(
-          "pandas.read_sql", return_value=expected,
-        ):
-        returned_samples = get_cherrypicked_samples(samples)
-        assert returned_samples.at[0, 'Root Sample ID'] == "MCM001"
-        assert returned_samples.at[1, 'Root Sample ID'] == "MCM003"
-        assert returned_samples.at[2, 'Root Sample ID'] == "MCM005"
+    expected = pd.DataFrame(
+        ["MCM001", "MCM003", "MCM005"], columns=["Root Sample ID"], index=[0, 1, 2]
+    )
+    samples = ["MCM001", "MCM002", "MCM003", "MCM004", "MCM005"]
+    plate_barcodes = ["123", "456"]
+
+    with app.app_context():
+        with patch("sqlalchemy.create_engine", return_value=Mock()):
+            with patch(
+                "pandas.read_sql", return_value=expected,
+            ):
+                returned_samples = get_cherrypicked_samples(samples, plate_barcodes)
+                assert returned_samples.at[0, "Root Sample ID"] == "MCM001"
+                assert returned_samples.at[1, "Root Sample ID"] == "MCM003"
+                assert returned_samples.at[2, "Root Sample ID"] == "MCM005"
+
```

Comment:
Not sure what this test is testing? Looks like its testing that the mock returns what its been told to return, rather than testing the SQL query is doing what it should. But if the SQL query gets pulled out into its own method then maybe that can be tested there instead.